modelina-website/src/helpers/GeneratorCode/CSharpGenerator.ts

```diff
@@ -17,6 +17,11 @@ export function getCSharpGeneratorCode(
     );
   }
 
+  if (generatorOptions.csharpOverwriteHashcode) {
+    optionString.push(
+      `   OverwriteHashcodeSupport : ${generatorOptions.csharpOverwriteHashcode}`
+    );
```

Comment:
this is incorrect, to add overwrite hashcode, it needs to be added as preset, i.e.

```
optionStringPresets.push(`
    {
      preset: CSHARP_COMMON_PRESET,
      options: {
        equal: false,
        hashCode: true
      }
    }`)
```
so you have to move this check a bit down.